src/mammos_entity/io.py

```diff
@@ -537,16 +595,23 @@ def _entities_from_yaml(filename: str | Path) -> EntityCollection:
     if not file_content["metadata"] or "version" not in file_content["metadata"]:
         raise RuntimeError("File does not have a key metadata:version.")
 
-    if (version := file_content["metadata"]["version"]) != "v1":
+    if (version := file_content["metadata"]["version"]) not in ["v1", "v2", "v3"]:
```

Comment:
- With my suggestion this will change to v1 or v2
- Otherwise we should not allow v2 if it has never been defined.